src/apps/search/list/Hits.tsx

```diff
@@ -0,0 +1,143 @@
+import {useHits} from 'react-instantsearch';
+import ListHit from '@components/custom/project/ListHit'
+import {useTranslations} from "@i18n/useTranslations";
```

Comment:
The `useTranslation` hook will make a `fetch` request to the API to retrieve the translations from Tina. This should only be done once at the most top level component. We can then use `TranslationContext` to supply the translation values to the component further down the component hierarchy.